packages/airnode-node/src/workers/local-gateways/validation.test.ts

```diff
@@ -229,17 +229,25 @@ describe('dropInconsistentBeacons', () => {
     [[0], [0]],
     [[], []],
     [[1, 2, 3], []],
+    [
+      [99, 99, 99, 101, 101, 101],
+      [99, 99, 99, 101, 101, 101],
+    ],
+    [
+      [99, 101, 101, 101, 101, 101, 101],
+      [99, 101, 101, 101, 101, 101, 101],
```

Comment:
This is not correct. The AVG = 100.714285714,  MIN = 99.7071428569

Is this because of flooring?